common/duckdb_logica.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+#
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import getpass
+import json
+import os
+import re
+from decimal import Decimal
+import sqlite3
+
+if '.' not in __package__:
+  from type_inference.research import infer
+else:
+  from ..type_inference.research import infer
+
+
+def PostgresExecute(sql, connection):
```

Comment:
This looks like a copy of postgres file. Do we need it at all?

Reply:
I think we don't need that and I will remove them.